### ISSUE TITLE
Internal Links: Titles with underscore in the name are escaped erroneously

### ISSUE THREAD

**Issue by finn-matti** [akosbalasko/yarle#90]
If your note has an underscore like this:

![Screenshot 2020-11-02 at 19 22 10](https://user-images.githubusercontent.com/56148301/97898584-b6633700-1d40-11eb-8cde-9dbb8a27be5d.png)

And another note links to it like this:

![Screenshot 2020-11-02 at 19 22 32](https://user-images.githubusercontent.com/56148301/97898616-c3802600-1d40-11eb-8d09-ac3b899310a6.png)

Then yarle will create markdown files with escaped underscores:

```md
# test - title_with_underscore

.

(Quelle: )

*   Zurück: 
    *   s
*   Weiter:
    *    s
*   Querverweise:
    *   s

    Created at: 2020-11-02T19:20:07+02:00
    Updated at: 2020-11-02T19:20:29+02:00
    Where: 29.18829748628224,65.96317843142039
```

```md
# test - title_with_underscore2

[[test - title\_with\_underscore]]

(Quelle: )

*   Zurück:
    *   s
*   Weiter:
    *   s
*   Querverweise:
    *   s

    Created at: 2020-11-02T19:20:32+02:00
    Updated at: 2020-11-02T19:20:58+02:00
    Where: 29.18829748628224,65.963134765625
```

Other symbols might also get escaped. I haven't checked.

This is somewhat related to #73

**Comment by finn-matti** [comment]
Okay. Some more escapes that are problematic:

```md
# test - title_with_underscore2

[[test - title\_with\_underscore]]

[[test - title\_with=+\*'-ö0=?lots of symbols/\\/||//\\\\(){}<>#\`\`´´^°]]

(Quelle: )

*   Zurück:
    *   s
*   Weiter:
    *   s
*   Querverweise:
    *   s

    Created at: 2020-11-02T19:20:32+02:00
    Updated at: 2020-11-02T19:39:05+02:00
    Where: 29.18829748628224,65.963134765625


```

 My assumption is that this is caused by [turndown's escapes](https://github.com/domchristie/turndown/blob/8ba04467c5fd73f58a1b49f68a66266f30fa202b/src/turndown.js#L9).

**Comment by akosbalasko** [comment]
implemented in #94
